Xamarin.PropertyEditing.Mac/HostResourceProvider.cs

```diff
@@ -13,6 +13,13 @@ public NSAppearance CurrentAppearance
 			set;
 		}
 
+		private readonly NSBundle resourceBundle;
+
+		public HostResourceProvider ()
+		{
+			this.resourceBundle = new NSBundle ("../../../PropertyEditingResource.bundle");
```

Comment:
We should change this to something like:
```
var containingDir = Path.GetDirectoryName (typeof (HostResourceProvider).Assembly.Location);
var bundle = Path.Combine (containingDir, "PropertyEditingResource.bundle");
```

Reply:
@alanmcgovern `HostResourceProvider`'s assembly ends up in `*.app/Contents/MonoBundle`, so we'd still need to `../../../` to get to the `Resource.bundle`. Should the copy put in the files in `*.app/Contents/MonoBundle` too, or not?

Reply:
Yup! We probably still need a `cp` in standalone designer to copy the bundle from the output directory inside the app bundle itself. I missed that in my original gist!